src/js/rendering/FramesheetRenderer.js

```diff
@@ -13,11 +13,13 @@
     }
   };
 
-  ns.FramesheetRenderer.prototype.renderAsCanvas = function () {
-    var canvas = this.createCanvas_();
+  ns.FramesheetRenderer.prototype.renderAsCanvas = function (framesPerRows) {
```

Comment:
PiskelRenderer and FramesheetRenderer are used by other callers than PngExport. 
framesPerRow should be defaulted to 0 if it's not provided.

(not related : I need to open another issue to get rid of "PiskelRenderer". Unnecessary and confusing ... congratulations for getting through this!)